bundle/manifests/sriov-network-operator.clusterserviceversion.yaml

```diff
@@ -222,6 +222,14 @@ spec:
           - '*'
           verbs:
           - '*'
+        - apiGroups:
```

Comment:
why do we still update this file which doesn't exist upstream?

Reply:
This was updated by ```make bundle```. @pliurh Any idea why there is a discrepancy?

Reply:
The bundle folder is downstream only. The upstream folder is a legacy which are not used any more.

Reply:
I hope we answered your question Zenghui.